src/styles/main.scss

```diff
@@ -1,3 +1,80 @@
+$indent-day: 1px;
+$size-day: 100px;
+$size-calendar: ($indent-day * 6) + (7 * $size-day);
+$week-days: (
+  'mon': 0,
+  'tue': 1,
+  'wen': 2,
+  'thu': 3,
+  'fri': 4,
+  'sut': 5,
+  'sun': 6
+);
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
 }
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 20px;
+  width: $size-calendar;
+  gap: $indent-day;
+
+  &__day {
+    box-sizing: border-box;
```

Comment:
Move it to the `*` selector
```suggestion
```

Reply:
But it is written in the   [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

<img width="569" alt="Don't use selector" src="https://github.com/mate-academy/layout_calendar/assets/155663344/09a72c86-3653-472b-84cc-94e133c10d0d">

Reply:
@Math8585 

You shouldn't use `*` selector for `margin` and `padding`, because it impacts the performance. 

But what about `box-sizing`? 🤔

In case you have 30 different blocks on the page, will you write `box-sizing` to all these blocks? In a big project if somebody forgot to add `box-sizing` it can be hard to debug in the future.

So it's good to write `box-sizing` in `*` selector.